devel/jsoncpp/Portfile

```diff
@@ -9,7 +9,7 @@ PortGroup           compiler_blacklist_versions 1.0
 compiler.cxx_standard 2011
 
 # some older Clang say they support C++11 when they don't
-compiler.blacklist-append {clang < 900}
+compiler.blacklist-append {clang < 800} {macports-clang-3.[0-8]}
```

Comment:
this needs to be `< 900` ... I just tested ...
```
$ clang -v
Apple LLVM version 8.0.0 (clang-800.0.42.1)
Target: x86_64-apple-darwin15.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```
and here's the logfile for your verification ...
[jsoncpp_clang_800.log.txt](https://github.com/macports/macports-ports/files/4476897/jsoncpp_clang_800.log.txt)

if I'm missing something, please tell me what to check or test or change & I'll try it out.

NOTE: This is an actual MacBook running OSX 10.11 ...

Reply:
Ehi thanks @michaellass, so the blacklist of the `macports-clang` up to 3.8 version is not needed?

Reply:
that fits .... clang 800 is approximately macports-clang-3.8, clang 900 = macports-clang-3.9.

so blacklisting clang < 900 means about the same as blacklisting macports-clang-3.[0-8].